plotters/src/style/color.rs

```diff
@@ -128,31 +114,40 @@ impl Color for RGBColor {
         }
     }
 }
+
 impl BackendStyle for RGBColor {
     fn color(&self) -> BackendColor {
         self.to_backend_color()
     }
 }
 
-/// The color described by HSL color space
 #[derive(Copy, Clone, PartialEq, Debug, Default)]
 #[cfg_attr(feature = "serialization", derive(Serialize, Deserialize))]
 pub struct HSLColor(pub f64, pub f64, pub f64);
 
+impl HSLColor {
+    #[inline]
+    pub fn from_degrees(h_deg: f64, s: f64, l: f64) -> Self {
```

Comment:
Perhaps return a `Result<HSLColor, some kind of error>` here
This way you guarantee that hue, saturation and light are all contained in [0,1]